src/components/Auth/TodoRender/TodoRender.tsx

```diff
@@ -0,0 +1,194 @@
+import classNames from 'classnames';
+import {
+  KeyboardEvent, useContext, useEffect, useRef,
+} from 'react';
+import { deleteTodo, updateTodo } from '../../../api/todos';
+import { TypeChange } from '../../../types/TypeChange';
+import { Todo } from '../../../types/Todo';
+import { TodoContext } from '../../../context/TodoContext';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoRender: React.FC<Props> = ({
+  todo,
+}) => {
+  const {
+    handleStatusChange,
+    selectedTodoId,
+    setSelectedTodoId,
+    setInputValue,
+    inputValue,
+    handleChangeTitle,
+    setLoadError,
+    setErrorMessage,
+    allCompletedLoader,
+    todoIdLoader,
+    setTodoIdLoader,
+    toggleLoader,
+  } = useContext(TodoContext);
+
+  const {
+    title, completed, id,
+  } = todo;
+  const newTodoField = useRef<HTMLInputElement>(null);
+
+  const removeFromServer = async (data: Todo) => {
+    try {
+      setTodoIdLoader(data.id);
+      await deleteTodo(data.id);
+      handleStatusChange(data, TypeChange.delete);
+    } catch (_) {
+      setLoadError(true);
+      setErrorMessage('Unable to delete todo from the server');
+    } finally {
+      setTodoIdLoader(null);
+      setSelectedTodoId(-1);
+    }
+  };
+
+  const handleRemoveTodo = (data: Todo) => {
+    removeFromServer(data);
+  };
```

Comment:
I'm sure that you can use removeFromServer instead of handleRemoveTodo. Don’t create unnecessary methods